app.py

```diff
@@ -0,0 +1,49 @@
+import fastapi, pydantic  # import libraries
+
+app = fastapi.FastAPI() # Initialize application
+
+
+
+class _data(pydantic.BaseModel): # create data model
+    text: str
+
+
+    class Config:
+        schema_extra = {
+            "example": {
+                "test": "secret text with password",
+            }
+        }
+
+
+@app.post("/parser/") # define endpoint name
+async def parserHandler(data: _data): # Handler to handle data
+    r = await parser(data)
+    return r
+
+async def parser(d: _data) -> str:
+    """_summary_
+    Takes data and returns formated response
+    Args:
+        d (_data): _description_
+
+    Returns:
+        str: _description_
+    """
+    import json
+    try:
+        with open('words.json') as j: # parse file
+            data = json.load(j)
+            data = data["words"]
+            count = 0
+            response = d.text.split(" ")
+            for i in d.text.split(" "):
+                for j in data:
+                    if i == j: # business logic
+                        response[count] = "*****"
+                    else:
+                        response[count] = i
```

Comment:
this whole loop is simple enough to turn into a comprehension: 
`[word for word in response if word not in passwords_set else REDACTED]`
not everything should be a comprehension though: https://google.github.io/styleguide/pyguide.html#27-comprehensions--generator-expressions

Reply:
even better if you use a generator comprehension because it uses less memory.